doc/README.md

```diff
@@ -1,7 +1,29 @@
 # Installation for OSX Development
+TODO(raj): Write a script to do all this
+
 - Install gcc on your Mac via running ```gcc``` on the terminal
 - This will prompt you to install XCode
 - Install XCode
 - Verify gcc was installed by running ```gcc```
+- Install git (search for it online)
+- Run ```git clone https://github.com/supragma/supragma.git```
+- Run inserting your username and email (instead of Raj's):
+```
+  git config --global user.name "Navraj Chohan"
+  git config --global user.email nlake44@gmail.com

```

Comment:
Do you want your personal email in a public repo?

Reply:
Yeah spammers already have it.